docs/admin/index.md

```diff
@@ -1,8 +1,14 @@
 # DTS Administrator Guide
 
+This guide walks you through the process of installing and configuring the DTS,
+both locally and in a Docker-based environment. It also discusses some technical
+issues that arise in granting the DTS permissions to filesystems and other
+services it needs to perform its data transfers.
```

Comment:
minor: I would just say "...services it needs to perform data transfers", but YMMV.